README.md

```diff
@@ -26,8 +26,7 @@ Just add `antigen bundle frmendes/geometry` to your `.zshrc`.
 
 ### oh-my-zsh
 
-Move `geometry.zsh` to `$HOME/.oh-my-zsh/custom/themes/geometry.zsh-theme` and
-set `ZSH_THEME="geometry"` in your `.zshrc`.
+Move the entire `geometry` folder to `$HOME/.oh-my-zsh/custom/themes`. Rename `geometry.zsh` to `geometry.zsh-theme` and set `ZSH_THEME="geometry/geometry"` in your `.zshrc`.
```

Comment:
`/Users/<name>/.oh-my-zsh/oh-my-zsh.sh:source:110: no such file or directory: /Users/<name>/.oh-my-zsh/themes/geometry.zsh-theme`

I tried it both on my mac & linux comps and both times I had to use zsh-theme.